canonical-websites/www.ubuntu.com
# [UA4I] 4797 openstack features

## Done

- updated storage section

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/openstack/features](http://0.0.0.0:8001/openstack/features)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- compare to the [copy doc](https://docs.google.com/document/d/1odFXuFtN1GjuZvgd8tW48HB4J9PjZJtP3w6wY0xlmog/edit#)


## Issue / Card

Fixes #4797
